deploy/template.yaml

```
@@ -766,6 +766,54 @@ Resources:
       FilterPattern: ""
       LogGroupName: !Ref IPVJourneyEngineFunctionLogGroup
 
+  IPVValidateCriCheckFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "validate-cri-check-${Environment}"
+      Handler: uk.gov.di.ipv.core.validatecricheck.ValidateCriCheckHandler::handleRequest
+      Runtime: java11
+      PackageType: Zip
+      CodeUri: ../lambdas/validatecricheck
+      Architectures:
+        - x86_64
+      MemorySize: 512
+      Tracing: Active
+      Environment:
+        Variables:
+          ENVIRONMENT: !Ref Environment
+          POWERTOOLS_SERVICE_NAME: !Sub validate-cri-check-${Environment}
+          USER_ISSUED_CREDENTIALS_TABLE_NAME: !Ref UserIssuedCredentialsTable
+      Policies:
+        - DynamoDBReadPolicy:
+            TableName: !Ref UserIssuedCredentialsTable
+      Events:
+        IPVCoreInternalAPI:
```

Think this line needs changing as well?
```suggestion
        IPVCorePrivateAPI:
```

---

Bah. Have fixed up. Thanks!
Just checking in with Dan about the open-api gateway spec. I can see the external one being referenced in the template, but not the internal one for the private API.